api/lib/domain/models/CertificationCenterMembership.js

```diff
@@ -21,11 +26,19 @@ class CertificationCenterMembership {
     this.updatedAt = updatedAt;
   }
 
+  get hasAdminRole() {
+    return this.role === CERTIFICATION_CENTER_MEMBERSHIP_ROLES.ADMIN;
+  }
+
+  get hasMemberRole() {
+    return this.role === CERTIFICATION_CENTER_MEMBERSHIP_ROLES.MEMBER;
+  }
```

Comment:
Pour les _Organizations_, dans le modèle `lib/domain/models/Membership.js` on a uniquement un getter `isAdmin` et le fait que le compte en question est « simple membre » est déduit par l'expression `!isAdmin`.

Dans le contexte du modèle de données actuel, je serais assez gêné par des fonctions de notre code qui vérifieraient si l'utilisateur « est Membre » car on ne saurait alors pas exactement si on veut vérifier si l'utilisateur « est juste un membre, admin ou pas » ou si on veut vérifier si l'utilisateur « est un simple membre et pas un admin ».

Aussi le code ci-dessous me semble préférable, qui est sur le même principe que pour les _Organizations_, car il lève cette ambiguïté : 

```suggestion
  get isAdmin() {
    return this.organizationRole === CERTIFICATION_CENTER_MEMBERSHIP_ROLES.ADMIN;
  }
```